data/shortcuts.json

```diff
@@ -85,9 +81,8 @@
               "text": "shortcuts.browsing.display_options.minimap"
             },
             {
-              "modifiers": ["⇧"],
-              "shortcuts": ["map_data.highlight_way_edits.key"],
-              "text": "shortcuts.browsing.display_options.highlight_way_edits",
+              "shortcuts": ["map_data.highlight_edits.key"],
```

Comment:
This change was not strictly a thing I fixed for this functionality- it was a mistake that I munged while merging from master. This puts it back where it belongs.